gui_windows.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2014 The gocui Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build windows
+
+package gocui
+
+import (
+	"os"
+	"syscall"
+	"unsafe"
+)
+
+type wchar uint16
+type short int16
+type dword uint32
+type word uint16
+
+type coord struct {
+	x short
+	y short
+}
+
+type smallRect struct {
+	left   short
+	top    short
+	right  short
+	bottom short
+}
+
+type consoleScreenBufferInfo struct {
+	size              coord
+	cursorPosition    coord
+	attributes        word
+	window            smallRect
+	maximumWindowSize coord
+}
+
+var (
+	kernel32                       = syscall.NewLazyDLL("kernel32.dll")
+	procGetConsoleScreenBufferInfo = kernel32.NewProc("GetConsoleScreenBufferInfo")
+)
+
+// getTermWindowSize is get terminal window size on windows.
+func (g *Gui) getTermWindowSize() (int, int, error) {
+	var csbi consoleScreenBufferInfo
+	r1, _, err := procGetConsoleScreenBufferInfo.Call(os.Stdout.Fd(), uintptr(unsafe.Pointer(&csbi)))
```

Comment:
Why not add the procGetConsoleScreenBufferInfo var above this function?  
```go
procGetConsoleScreenBufferInfo := syscall.NewLazyDLL("kernel32.dll").NewProc("GetConsoleScreenBufferInfo")
r1, _, err := procGetConsoleScreenBufferInfo.Call(os.Stdout.Fd(), uintptr(unsafe.Pointer(&csbi)))
```

Reply:
I rarely look your style.

Reply:
Sure then we will leaf it like this.